docs/pages/application-access/cloud-apis/aws-console.mdx

```diff
@@ -269,44 +527,136 @@ AWS roles in these accounts.
+   once.)
+
+1. Click the **Launch** button for the AWS Console application, then choose the
+   role you would like to assume when signing in to the AWS Console:
+
+   ![IAM role selector](../../../img/application-access/iam-role-selector.png)
 
-![IAM role selector](../../../img/application-access/iam-role-selector.png)
+1. Click on the role you want to assume and you will get redirected to the AWS
+   Management Console, signed in with the selected role. You should see your
+   Teleport user name as a federated login assigned to `ExampleReadOnlyRole` in
+   the top-right corner of the AWS Console:
 
-Click on the role you want to assume and you will get redirected to the AWS
-Management Console, signed in with the selected role.
+   ![Federated login](../../../img/application-access/federated-login@2x.png)
 
-In the console's top-right corner you should see that you're logged in through
-federated login and the name of your assumed IAM role:
+### Access the AWS CLI
 
-![Federated login](../../../img/application-access/federated-login@2x.png)
+1. Log into the previously configured AWS console app on your desktop:
 
-Note that your federated login session is marked with your Teleport username.
+   ```code
+   $ tsh apps login --aws-role ExampleReadOnlyAccess awsconsole
+   Logged into AWS app "awsconsole".
+   
+   Your IAM role:
+     arn:aws:iam::000000000000:role/ExampleReadOnlyAccess
+   
+   Example AWS CLI command:
+     tsh aws s3 ls
+   
+   Or start a local proxy:
+     tsh proxy aws --app awsconsole
+   ```
 
-<Admonition type="note" title="Session Duration">
-    Due to limitations in AWS IAM [role
-    chaining](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_terms-and-concepts.html#iam-term-role-chaining),
-    Management Console sessions will be restricted to one hour if Teleport is
-    running with temporary security credentials (e.g., when using IAM roles for
-    service accounts on EKS).
-</Admonition>
+   The `--aws-role` flag allows you to specify the AWS IAM role to assume when
+   accessing AWS APIs. You can either provide a role name like `--aws-role
+   ExampleReadOnlyAccess` or a full role ARN like
+   `arn:aws:iam::0000000000:role/ExampleReadOnlyAccess`.
+
+1. Now you can use the `tsh aws` command like the native `aws` command-line
+   tool:
+
+   ```code
+   $ tsh aws s3 ls
+   ```
+
+1. Run the following command to log out of the AWS application and remove
+   credentials:
+
+   ```code
+   $ tsh apps logout awsconsole
+   ```
+
+### Access applications using AWS SDKs
+
+1. Log into the AWS Management Console app you configured earlier:
+
+   ```code
+   $ tsh apps login --aws-role ExampleReadOnlyAccess awsconsole
+   ```
+
+1. Use the following command to start a local HTTPS proxy server, which forwards
+   AWS API traffic to the Teleport Application Service. Leave the terminal where
+   you run the command open since it runs in the foreground:
```

Comment:
```suggestion
   you ran the command open since it runs in the foreground:
```

Reply:
Suggested a clearer wording since "run" and "ran" can be ambiguous in this paragraph.